src/Pages/Form/Components/ChiefComplaintSection.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { ChangeEvent, useState } from "react";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import {
+  createStyles,
+  makeStyles,
+  createTheme,
+  Theme,
+  ThemeProvider,
+  withStyles,
+} from "@material-ui/core/styles";
+import {
```

Comment:
Please extract all imports to default imports(eg `import Box from '@material-ui/core/Box'`). Because importing object is importing all of component in @material-ui/core.